Monika After Story/game/script-topics.rpy

```diff
@@ -14269,3 +14269,55 @@ label monika_hot_springs:
     m 5dkbfb "Ahhh,{w=0.2} {nw}"
     extend 5dkbfa "just the thought of it makes me feel all tingly, [player]~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_using_computers_healthily",
+            category=['advice','life', 'you'],
+            prompt="Using computers healthily",
+            random=True,
+            aff_range=(mas_aff.AFFECTIONATE, None)
```

Comment:
Not sure this topic needs an aff range at all, but if it does, it should be Normal+

Reply:
Amended. I've kept the affection range because I wanted to tie in the start of the conversation with an appropriate nice gesture from Monika at the end; it wouldn't really work without a positive relationship with the player imo.